src/styles/blocks/card.scss

```diff
@@ -0,0 +1,101 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+
+}
+
+p {
+  margin: 0;
+}
+
+.product {
+  width: 200px;
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column; /* Stacks items vertically */
+  justify-content: space-between;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+  }
+
+  &__description {
+    padding-top: 40px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $Main-accent;
+  }
+
+  &__code {
+    padding-top: 4px;
+    font-weight: 400;
```

Comment:
it's default value, so we can remove it

Reply:
That's not default values, if remove them, will ocure mistake. Other mistakes were fixed